.travis.yml

```diff
@@ -50,7 +49,7 @@ jobs:
       <<: *disable_global
       stage: unsupported
       script: .misc/check_unsupported.sh
-    - python: 3.4.3
+    - python: 3.4.1
```

Comment:
can you pull this change out into a separate commit , and I'd like to see a build with only this commit so I can see if this solves the breakage.  if not, why not.

Reply:
As I expected, the job is green after this fix.  Please make a separate PR for just that change, as that is what solves the issue and turns our CI for master green. (the min pip problem is a separate problem which doesnt need to be solved urgently)

The commit message should indicate that the bug is in my commit which dropped the minimum version but forgot to update .travis.yml